apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx

```diff
@@ -369,21 +354,20 @@ export const CreateListingsForm: React.FC<NFTMintForm> = ({
           </Stack>
         ) : null}
       </FormControl>
-      {contractType === "marketplace" ? (
-        <FormControl isDisabled={noNfts}>
-          <Heading as={FormLabel} size="label.lg">
-            Listing Type
-          </Heading>
-          <Select {...form.register("listingType")}>
-            <option value="direct">Direct</option>
-            <option value="auction">Auction</option>
-          </Select>
-          <FormHelperText>
-            The type of listing you want to create, either an auction or direct
-            listing.
-          </FormHelperText>
-        </FormControl>
-      ) : null}
+      {/* Just don't support Marketplace v2 anymore */}
```

Comment:
please make sure support is aware of this change in case we start hearing about it

Reply:
Noticed them